profiler/build.gradle.kts

```diff
@@ -7,6 +7,7 @@ dependencies {
   compileOnly("io.opentelemetry:opentelemetry-semconv")
   implementation("io.opentelemetry.proto:opentelemetry-proto")
   implementation("io.opentelemetry:opentelemetry-sdk-logs")
+  implementation("io.opentelemetry:opentelemetry-exporter-otlp-logs")
```

Comment:
can we now remove `opentelemetry-proto` and `grpc` dependencies or are they still needed?

Reply:
Good call. We don't need those, but there was a hidden transitive dep in the test code on `com.google.common.io.CharStreams` but I was able to remove that and allow the tests to run with java 11, which is a win-win.